CODE_OF_CONDUCT.md

```diff
@@ -22,23 +72,23 @@ Examples of unacceptable behavior by participants include:
 * Publishing others' private information, such as a physical or electronic address, without explicit permission
 * Other conduct which could reasonably be considered inappropriate in a professional setting
 
-## Our Responsibilities
+### Our Responsibilities
 
 Project maintainers are responsible for clarifying the standards of acceptable behavior and are expected to take appropriate and fair corrective action in response to any instances of unacceptable behavior.
 
 Project maintainers have the right and responsibility to remove, edit, or reject comments, commits, code, wiki edits, issues, and other contributions that are not aligned to this Code of Conduct, or to ban temporarily or permanently any contributor for other behaviors that they deem inappropriate, threatening, offensive, or harmful.
 
-## Scope
+### Scope
 
 This Code of Conduct applies both within project spaces and in public spaces when an individual is representing the project or its community. Examples of representing a project or community include using an official project e-mail address, posting via an official social media account, or acting as an appointed representative at an online or offline event. Representation of a project may be further defined and clarified by project maintainers.
 
-## Enforcement
+### Enforcement
 
 Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting the project team at info@ml5js.org. The project team will review and investigate all complaints, and will respond in a way that it deems appropriate to the circumstances. The project team is obligated to maintain confidentiality with regard to the reporter of an incident. Further details of specific enforcement policies may be posted separately.
```

Comment:
Just noting the info@ml5js.org address goes to me. I recently read a great thread (having trouble finding it at the moment) on Twitter about how reporting should always have more than one point of contact (in case the report is about the point of contact). Maybe I can have that e-mail forward to multiple addresses or we can include hello@ml5js.org as well? Should we be transparent about where those addresses forward to?

Reply:
Yes - I think it make sense to write:
"... email Dan @ info@ml5js.org or Ashley  @ hello@ml5js.org..."

Something like this?

@shiffman @AshleyJaneLewis  - would this be ok for you?

Reply:
Yes, that works for me! I can imagine that we would change where these addresses forward to from time to time, but probably good to be as clear as possible for now. We can make a mental note to try to keep this up to date as we go forward. . .

Reply:
Cool. Sounds good to me. 

On bigger note of devOps related topics, I wonder if we can set up something where we can include a `config.json` file that includes details like:

```json
{
"contact":{
   "info":{
      "email":"info@ml5js.org",
      "name":"dan shiffman"
    },
   "hello":{
       "email":"hello@ml5js.org",
      "name":"ashley"
    }
}
}
```


then in our docs something like:

```md

... reach out to {{config.contact.info.name}} at {{config.contact.info.email}} for details about XYZ.
```

Maybe something that could be part of keeping our docs up to date and easier to manage :)

Reply:
Yes, this seems like a great idea!